python/chemiscope/explore.py

```diff
@@ -30,6 +32,12 @@ def explore(frames, featurize=None, properties=None, mode="default"):
         with the atomic structures. Properties can be extracted from frames with
         :py:func:`extract_properties` or manually defined by the user.
 
+    :param environments: optional. List of environments (described as
+        ``(structure id, center id, cutoff)``) to include when extracting the
+        atomic properties. Can be extracted from frames with
+        :py:func:`all_atomic_environments` (or :py:func:`librascal_atomic_environments`)
+        or manually defined.
+
```

Comment:
if the user gives a subset of environments, we need to make sure the per-atom features are only defined for these specific atom-centered environments. I would do this in a separate PR, since it is orthogonal to metatensor models.

Reply:
This can be done with the `_normalize_environments` as it was implemented here ?

https://github.com/lab-cosmo/chemiscope/blob/49b1182a9f97a22fed7e4918148e0620bda5c0b7/python/chemiscope/input.py#L310-L315